Formula/genext2fs.rb

```diff
@@ -0,0 +1,19 @@
+class Genext2fs < Formula
+  desc "Generates an ext2 filesystem as a normal (non-root) user"
+  homepage "http://freecode.com/projects/genext2fs"
```

Comment:
Freecode is dead. Might as well use the SF homepage at http://genext2fs.sourceforge.net/